site/vite.config.mts

```diff
@@ -92,6 +92,7 @@ export default defineConfig({
 			testHelpers: path.resolve(__dirname, "./src/testHelpers"),
 			theme: path.resolve(__dirname, "./src/theme"),
 			utils: path.resolve(__dirname, "./src/utils"),
+			"@": path.resolve(__dirname, "./src"),
```

Comment:
🙅‍♀️

Reply:
same here, this is just here in attempts to use the shadcn cli